server/src/main/java/org/elasticsearch/common/collect/EvictingQueue.java

```diff
@@ -1,20 +1,17 @@
-/*
- * Licensed to Elasticsearch under one or more contributor
- * license agreements. See the NOTICE file distributed with
- * this work for additional information regarding copyright
- * ownership. Elasticsearch licenses this file to you under
- * the Apache License, Version 2.0 (the "License"); you may
- * not use this file except in compliance with the License.
+/* @notice
+ * Copyright (C) 2012 The Guava Authors
```

Comment:
One challenge here is that we *have* made modifications to these files, and my understanding is that we're suppose to indicate that. That is, this is *not* a direct copy of EvictingQueue from Guava, but rather the starting point, and then we made some modifications in place which are of course not copyright the Guava authors.

Reply:
Same situation for our Guice fork. What's the right course of action here? Do we require a slightly different header, or include both the ES one _and_ the vendored one?

Reply:
@jasontedor can we push forward on this as-is or do you suspect we need to change these headers?

@tomcallahan do you have thoughts on the proper way to handle vendored code that is not a direct copy, but _has_ been modified?

Reply:
@mark-vieira - @jasontedor is correct that we should flag the fact that we've modified these files. I think it would be fine to simply add something like this:

`Modifications copyright (C) <year> Elasticsearch B.V.`

after the Guava copyright